server/sequelize.js

```diff
@@ -5,6 +5,15 @@ import EncryptedField from "sequelize-encrypted";
 
 const isProduction = process.env.NODE_ENV === "production";
 
+const isPGSqlEnabled =
```

Comment:
I think checking for the explicit disable of `process.env.PGSSLMODE !== "disable"` should be enough, no?